miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/SamplePage.java

```diff
@@ -0,0 +1,501 @@
+  @FindBy(id = "detailedQcStatus")
+  private WebElement qcStatusLabel;
+  @FindBy(id = "qcStatusNote")
+  private WebElement qcNoteLabel;
+  @FindBy(id = "volume")
+  private WebElement volumeLabel;
+  @FindBy(id = "discarded")
+  private WebElement discardedLabel;
+  @FindBy(id = "identityId")
+  private WebElement identityIdLabel;
+  @FindBy(id = "externalName")
+  private WebElement externalNameLabel;
+  @FindBy(id = "donorSex")
+  private WebElement donorSexLabel;
+  @FindBy(id = "parentAlias")
+  private WebElement parentAliasLabel;
+  @FindBy(id = "sampleClass")
+  private WebElement sampleClassLabel;
+  @FindBy(id = "sampleClassAlias")
+  private WebElement sampleClassAliasLabel;
+  @FindBy(id = "subProject")
+  private WebElement subProjectLabel;
+  @FindBy(id = "groupId")
+  private WebElement groupIdLabel;
+  @FindBy(id = "groupDescription")
+  private WebElement groupDescriptionLabel;
+  @FindBy(id = "concentration")
+  private WebElement concentrationLabel;
+  @FindBy(id = "tissueClass")
+  private WebElement tissueClassLabel;
+  @FindBy(id = "tissueOrigin")
+  private WebElement tissueOriginLabel;
+  @FindBy(id = "tissueType")
+  private WebElement tissueTypeLabel;
+  @FindBy(id = "passageNumber")
+  private WebElement passageNumberLabel;
+  @FindBy(id = "timesReceived")
+  private WebElement timesReceivedLabel;
+  @FindBy(id = "tubeNumber")
+  private WebElement tubeNumberLabel;
+  @FindBy(id = "externalInstituteIdentifier")
+  private WebElement externalInstituteIdentifierLabel;
+  @FindBy(id = "lab")
+  private WebElement labLabel;
+  @FindBy(id = "tissueMaterial")
+  private WebElement tissueMaterialLabel;
+  @FindBy(id = "region")
+  private WebElement regionLabel;
+  @FindBy(id = "slidesRemaining")
+  private WebElement slidesRemainingLabel;
+  @FindBy(id = "slides")
+  private WebElement slidesLabel;
+  @FindBy(id = "discards")
+  private WebElement discardsLabel;
+  @FindBy(id = "thickness")
+  private WebElement thicknessLabel;
+  @FindBy(id = "stain")
+  private WebElement stainLabel;
+  @FindBy(id = "slidesConsumed")
+  private WebElement slidesConsumedLabel;
+  @FindBy(id = "strStatus")
+  private WebElement strStatusLabel;
+  @FindBy(id = "DNAseTreated")
+  private WebElement dnaseTreatedLabel;
+  @FindBy(id = "samplePurpose")
+  private WebElement samplePurposeLabel;
+
+  public SamplePage(WebDriver driver) {
+    super(driver);
+    PageFactory.initElements(driver, this);
+    // oh this looks awful:
+    // https://stackoverflow.com/questions/18206578/java-lang-classcastexception-com-sun-proxy-proxy8-cannot-be-cast-to-org-openqa
+      
+    waitWithTimeout().until(titleContains("Sample "));
+  }
+
+  public static SamplePage get(WebDriver driver, String baseUrl, long sampleId) {
+    driver.get(baseUrl + "miso/sample/" + sampleId);
+    return new SamplePage(driver);
+  }
+
+  public String getId() {
+    return idLabel.getText();
+  }
+
+  public void setId() {
+    throw new UnsupportedOperationException();
```

Comment:
Initially, it was to remind myself that I hadn't forgotten them. I can delete them.